src/clusterfuzz/_internal/tests/core/bot/tasks/trials_test.py

```diff
@@ -13,7 +13,8 @@
 # limitations under the License.
 """Tests for app specific trials and experiments."""
 
-import unittest
+from pyfakefs import fake_filesystem_unittest
```

Comment:
You need to space it and order these imports like so:

```python
from pyfakefs import fake_filesystem_unittest

from clusterfuzz._internal.bot.tasks import trials
from clusterfuzz._internal.datastore import 
from clusterfuzz._internal.tests.test_libs import
```

Reply:
done